### DISCUSSION TITLE
被ChatGPT暖暈了

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#746] [General]
文章中寫到「Chat GPT不關心你最終決定吃什麼，只關心你的健康」
太霸氣了吧……
會幫你決定要吃什麼之餘還顧及營養均衡
還作詞寫曲
你的另一半說不定都沒有chat gpt浪漫……
真是一個有趣又實用的聊天機器人
如果可以講笑話或給予情緒上的支持
或是新增一個不定時關心你生活日常的功能
這樣的話不僅可以給使用者驚喜感 還可以讓用戶有抒發心事的感覺
那真的會讓人暈沒了……
https://www.cheers.com.tw/article/article.action?id=5101461

**Comment by jerry0108566** [comment]
我只會把Chat GPT當作寫心得報告的工具人..

**Reply by wu51000** [reply]
今晚我想來點
Chat gpt假男/女朋友……
（聽起來有點可悲……
